test/screenshot/screenshot.js

```diff
@@ -1,60 +1,213 @@
+import {Readable} from 'stream';
+import {createHash} from 'crypto';
+import {readFile, writeFile} from 'fs';
+import {promisify} from 'util';
 import puppeteer from 'puppeteer';
 import compareImages from 'resemblejs/compareImages';
-import {promisify} from 'util';
-import {readFile, writeFile} from 'fs';
 import {assert} from 'chai';
+import Storage from '@google-cloud/storage';
 
 import comparisonOptions from './screenshot-comparison-options';
 
 const readFilePromise = promisify(readFile);
 const writeFilePromise = promisify(writeFile);
 
+const SERVICE_ACCOUNT_KEY = process.env.MDC_GCLOUD_SERVICE_ACCOUNT_KEY;
```

Comment:
dont need all caps constants

Reply:
Are you sure?

The [Google style guide](https://google.github.io/styleguide/jsguide.html#naming-constant-names) says that constants should be all uppercase.